elasticdl/pkg/ps/optimizer.go

```diff
@@ -235,8 +281,9 @@ const (
 )
 
 var optArgumentsMap = map[string][]string{
-	"SGD":  []string{optArgLR, optArgMomentum, optArgNesterov},
-	"Adam": []string{optArgLR, optArgBeta1, optArgBeta2, optArgEpsilon, optArgAmsgrad},
+	"SGD":     []string{optArgLR, optArgMomentum, optArgNesterov},
```

Comment:
MomentumOptimizer missing?

Reply:
The paraments MomentumOptimizer need are incuded in SGD already.  `optArgLR, optArgMomentum, optArgNesterov`